internal/plugins/manifests/init.go

```diff
@@ -105,5 +113,12 @@ bundle: kustomize
 .PHONY: bundle-build
 bundle-build:
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+`
+	makefileTestTarget = `# Run tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+test: generate fmt vet manifests
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/{{ .ControllerRuntimeEnvTestVersion }}/hack/setup-envtest.sh
```

Comment:
If I understand correctly, this file has already been scaffolded and we're writing this directly into that file.

Therefore, I'm confused about the `{{ .ControllerRuntimeEnvTestVersion }}` template. Is that getting filled in somehow?

I searched the SDK codebase and couldn't find any other references to it. I only see it in the kubebuilder v3 plugin (which I don't think we're using, and even if we were, it would have already ran by the time this substitution happens).

Just want to make sure I'm not missing something :)

Reply:
Good catcher. I missed that in the copy/past.